Makefile

```diff
@@ -37,6 +37,10 @@ populate: create_db  ## Build and run containers
 	## Populate database with test data
 	docker-compose run --rm web python ../test_data.py -p
 
+.PHONY: testlocal
```

Comment:
Why does phony have to be used here and not in the other project?

Reply:
Hmmm, based on this explanation I think we should add them to the other one because none of the targets we use ever represent files: https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile

Reply:
Ahhh, okay great to know